tests/core/ingestion/test_family.py

```diff
@@ -211,7 +215,6 @@ def test_family_document_from_row__updates_status(test_db: Session, patch_curren
         2000, 1, 1, 0, 0, 0, 0, tzinfo=timezone.utc
     )
     assert db_family_doc.last_modified is not None
-    assert db_family_doc.created != db_family_doc.last_modified
```

Comment:
For context I believe @diversemix likes this way of building up the tests into stronger assertions from a loose initial assertion, but not an issue!